test/Project.toml

```diff
@@ -14,13 +14,3 @@ RDatasets = "ce6b1742-4840-55fa-b093-852dadbb1d8b"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 StableRNGs = "860ef19b-820b-49d6-a774-d7a799459cd3"
```

Comment:
In my experience, having a separate test/Project.toml is problematic when `dev`ing several packages locally, and I moved away from this. Can we get rid of it altogether? Maybe this even fixes your LTS fail.